geocities.jp,"HTTPSConnectionPool(host='geocities.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd5507ee4f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='geocities.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd5507eedf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
geomedian.com,"HTTPSConnectionPool(host='geomedian.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='geomedian.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
gesetze-im-internet.de,"HTTPSConnectionPool(host='gesetze-im-internet.de', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd550748670>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='gesetze-im-internet.de', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd55070edc0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
gestionderiesgos.gob.ec,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
gesundheitsfrage.net,"HTTPSConnectionPool(host='www.gesundheitsfrage.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gesundheitsfrage.net', port=443): Read timed out. (read timeout=2)"
gesundheitsinformation.de,"HTTPSConnectionPool(host='gesundheitsinformation.de', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd5507488e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='gesundheitsinformation.de', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd5506bf040>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
getnews.jp,"HTTPSConnectionPool(host='getnews.jp', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='getnews.jp', port=443): Read timed out. (read timeout=2)"
geva.co.il,"HTTPSConnectionPool(host='geva.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.geva.co.il', port=443): Read timed out. (read timeout=2)"
gezinomi.com,"HTTPSConnectionPool(host='www.gezinomi.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gezinomi.com', port=443): Read timed out. (read timeout=2)"
ggwp.id,"HTTPSConnectionPool(host='ggwp.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ggwp.id', port=443): Read timed out. (read timeout=2)"
ghidulprimariilor.ro,"HTTPSConnectionPool(host='ghidulprimariilor.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='ghidulprimariilor.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
giancarlorizza.it,Exceeded 30 redirects.,Exceeded 30 redirects.
gids.tv,"HTTPSConnectionPool(host='www.gids.tv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gids.tv', port=443): Read timed out. (read timeout=2)"
gifu.lg.jp,"HTTPSConnectionPool(host='gifu.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd54fe12f70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='gifu.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd55041a4f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
gigabaza.ru,"HTTPSConnectionPool(host='gigabaza.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='gigabaza.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
gintsristamecs.lv,"HTTPSConnectionPool(host='gintsristamecs.lv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd54f793670>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='gintsristamecs.lv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd54f6f18b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
gismeteo.lt,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gismeteo.lv,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gismeteo.pl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gismeteo.ru,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gismeteo.ua,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
glasistre.hr,"HTTPSConnectionPool(host='glasistre.hr', port=443): Read timed out.","HTTPConnectionPool(host='glasistre.hr', port=80): Read timed out."
glaspodravine.hr,"HTTPSConnectionPool(host='www.glaspodravine.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.glaspodravine.hr', port=443): Read timed out. (read timeout=2)"
glassumadije.rs,"HTTPSConnectionPool(host='www.glassumadije.rs', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.glassumadije.rs', port=80): Read timed out. (read timeout=2)"
glimp.co.nz,"HTTPSConnectionPool(host='www.glimp.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.glimp.co.nz', port=443): Read timed out. (read timeout=2)"
gme.cz,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gnr.pt,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gobiernodecanarias.org,"HTTPSConnectionPool(host='gobiernodecanarias.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd54fd77460>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='gobiernodecanarias.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd54fdb4790>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
gogov.ru,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
goias.gov.br,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
goldcoastbulletin.com.au,"HTTPSConnectionPool(host='www.goldcoastbulletin.com.au', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.goldcoastbulletin.com.au', port=443): Read timed out. (read timeout=2)"
golos.com.ua,"HTTPSConnectionPool(host='golos.com.ua', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd550395a90>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='www.golos.com.ua', port=80): Read timed out. (read timeout=2)"
golvshop.se,"HTTPSConnectionPool(host='golvshop.se', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.golvshop.se', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
googleusercontent.com,"HTTPSConnectionPool(host='googleusercontent.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd54fc13df0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='googleusercontent.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd550068850>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
gorzow.pl,"HTTPSConnectionPool(host='gorzow.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd54f6fe6a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='gorzow.pl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd5501c5190>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
gov.bc.ca,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
gov.hu,"HTTPSConnectionPool(host='gov.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd5504fca30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='gov.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd550377550>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
govextra.gov.il,"HTTPSConnectionPool(host='govextra.gov.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='govextra.gov.il', port=443): Read timed out. (read timeout=2)"
gpoint.co.jp,"HTTPSConnectionPool(host='gpoint.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'gpoint.co.jp' doesn't match '*.gpoint.co.jp'"")))","HTTPSConnectionPool(host='gpoint.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'gpoint.co.jp' doesn't match '*.gpoint.co.jp'"")))"
grad-zadar.hr,"HTTPSConnectionPool(host='grad-zadar.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd550232400>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='grad-zadar.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd550d2a760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
grainger.ca,"HTTPSConnectionPool(host='grainger.ca', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='grainger.ca', port=80): Read timed out. (read timeout=2)"
grainger.com,"HTTPSConnectionPool(host='www.grainger.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.grainger.com', port=443): Read timed out. (read timeout=2)"
grainger.com.mx,"HTTPSConnectionPool(host='grainger.com.mx', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='grainger.com.mx', port=80): Read timed out. (read timeout=2)"
gramatnicaglobuss.lv,"HTTPSConnectionPool(host='gramatnicaglobuss.lv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gramatnicaglobuss.lv', port=443): Read timed out. (read timeout=2)"
gruene.at,"HTTPSConnectionPool(host='gruene.at', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='gruene.at', port=443): Read timed out. (read timeout=2)"
grupolusofona.pt,"HTTPSConnectionPool(host='grupolusofona.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd5504a2370>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='grupolusofona.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd54fcca4c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
gsb.or.th,"HTTPSConnectionPool(host='gsb.or.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd54ffadd00>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='gsb.or.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd54f82dd00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
gtp.gr,"HTTPSConnectionPool(host='gtp.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'gtp.gr' doesn't match either of 'www.gtp.gr', 'news.gtp.gr', 'traveloffers.gtp.gr', 'jobs.gtp.gr', 'about.gtp.gr', 'ferries.gtp.gr'"")))","HTTPSConnectionPool(host='gtp.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'gtp.gr' doesn't match either of 'www.gtp.gr', 'news.gtp.gr', 'traveloffers.gtp.gr', 'jobs.gtp.gr', 'about.gtp.gr', 'ferries.gtp.gr'"")))"
guaranteepra.com,"HTTPSConnectionPool(host='guaranteepra.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd550d2a0a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='guaranteepra.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd5503953d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
guiadelocio.com,"HTTPSConnectionPool(host='www.guiadelocio.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.guiadelocio.com', port=443): Read timed out. (read timeout=2)"
guiagt.com,"HTTPSConnectionPool(host='guiagt.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='guiagt.com', port=443): Read timed out. (read timeout=2)"
guiatelefonica.com.ec,"HTTPSConnectionPool(host='guiatelefonica.com.ec', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd54f7934f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='guiatelefonica.com.ec', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd54f7389d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
guiato.com.br,"HTTPSConnectionPool(host='guiato.com.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='guiato.com.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
guliveriokeliones.lt,"HTTPSConnectionPool(host='guliveriokeliones.lt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.guliveriokeliones.lt', port=443): Read timed out. (read timeout=2)"
gunadarma.ac.id,"HTTPSConnectionPool(host='gunadarma.ac.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.gunadarma.ac.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
guru.ua,"HTTPSConnectionPool(host='guru.ua', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='guru.ua', port=80): Read timed out. (read timeout=2)"
gutefrage.net,"HTTPSConnectionPool(host='www.gutefrage.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gutefrage.net', port=443): Read timed out. (read timeout=2)"
guzzle.co.za,"HTTPSConnectionPool(host='guzzle.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'guzzle.co.za' doesn't match 'www.guzzle.co.za'"")))","HTTPSConnectionPool(host='www.guzzle.co.za', port=443): Read timed out. (read timeout=2)"
gyorplusz.hu,"HTTPSConnectionPool(host='www.gyorplusz.hu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gyorplusz.hu', port=443): Read timed out. (read timeout=2)"
ha.rs,"HTTPSConnectionPool(host='ha.rs', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fd5506b4ac0>, 'Connection to ha.rs timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ha.rs', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fd55070e100>, 'Connection to ha.rs timed out. (connect timeout=2)'))"
haber365.com.tr,"HTTPSConnectionPool(host='haber365.com.tr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd54f7382e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='haber365.com.tr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd54f897f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
haifa.ac.il,"HTTPSConnectionPool(host='haifa.ac.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='haifa.ac.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
hala.jo,"HTTPSConnectionPool(host='hala.jo', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='hala.jo', port=80): Read timed out. (read timeout=2)"
